packages/elements/src/components/ino-carousel-slide/ino-carousel-slide.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, Element, Host, Prop, h, ComponentInterface } from '@stencil/core';
+import classNames from 'classnames';
+
+@Component({
+  tag: 'ino-carousel-slide',
+  styleUrl: 'ino-carousel-slide.scss',
+  shadow: false
+})
+export class InoCarouselSlide implements ComponentInterface{
+
+  @Element() el: HTMLElement;
+
+  /**
+   * Link to the image
+   */
+  @Prop() src: string;
+
+  /**
+   * Value of the slide
+   */
+  @Prop() value: any;
+
+  /**
+   * Indicates whether the slide is selected or not
+   */
+  @Prop() inoSelected: boolean = false;
```

Comment:
Should this be set by the consumer? If I understand correctly, this prop is only managed by the `ino-carousel`, isn't it? If that's the case, we should add it to the description or make sure that it cannot be set by the user (not sure how at the moment).

Reply:
We had a similar situation in the `ino-radio-group`. When the value changed, we overwrote the `value` property of every `ino-radio` but the selected. See the following: https://github.com/inovex/elements/blob/2c15cc8d44a2191ec4dd356f33e39ed75e7ae35f/packages/elements/src/components/ino-radio-group/ino-radio-group.tsx#L65

Reply:
We could also remove the property entirely and simply add the corresponding CSS class to the selected slide and remove it from the remaining slides